setup.cfg

```diff
@@ -80,3 +81,19 @@ include_trailing_comma = True
 force_grid_wrap = 0
 use_parentheses = True
 line_length = 88
+
+[coverage:run]
+dynamic_context = test_function
+include=
```

Comment:
This comes from the `$(COVERAGE) run "--include=$(PACKAGE)/*.py,$(TESTS_DIR)/*.py"...` in the `Makefile`

Reply:
Already addressed.